Contributing.md

```diff
@@ -0,0 +1,160 @@
+instance:
+
+- `789: Flow diagrams for PDF Font Embedder`
+
+### PR Labels
+
+All PRs should have at least *one* label, with the primary labels being:
+
+- `enhancement` for stories/spikes/tech-debt
+- `bug` for bugs
+- `documentation` for PRs which only include documentation changes
+
+### PR Descriptions
+
+The PR templates includes a number of sections:
+
+- *Issues*
+  - A link to the Issue(s) which this PR addresses should be included here
+- *Checklist*
+  - This includes a number of things which may need to be addressed for each PR
+  - If an item is accomplished (for instance, the PR was labeled appropriately)
+    modify the checked box for that item from `[ ]` to `[x]` to show that it has
+    been handled
+- *Notes for Reviewers*
+  - This section should give a high level description of the PR and how best to
+    inspect the changes
+- *Verification Instructions*
+  - This section should give step-by-step instructions for verifying that the
+    code changes are working
+
+### Merging
+
+Once the PR has been reviewed, approved & passes all automated checks, it can be merged; but the merge operation to use depends on the source and target branches of the pull request:
+
+Source Branch|Target Branch|Merge Operation|Notes
+-|-|-|-
+_any_|`main`|Merge|Preserves development history of release in the `main` branch
+hotfix branches: `hotfix/*` |_any_|Merge|Permits the same Hotfix to merge to multiple target branches
+development branches: `fix/*`, `feature/*`, `td/*`, `spike/*`|`dev`|Squash and merge|Before confirming, verify that the commit title is the same as the PR title, and the commit description is useful.
+
+## Branching
+
+Branches in this repository follow the [gitflow branching workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+### Development branches
+
+All development branches should follow the `<type>/<issue-id>/<description>` naming pattern; for example:
+
+- Stories: `feature/123/myStoryDescription`
+- Bugs: `fix/234/myBugDescription`
+- Spikes: `spike/345/mySpikeDescription`
+- Tech Debt: `td/456/myTechDebtDescription`
+
+In all cases, the final `<branchDescription>` piece should be written in camelCase.
+
+### dev
+
+The `dev` branch is the base branch from which all new feature branches are created.
+
+It is the default branch for feature and fix PRs to merge into, and always contains
+the most up-to-date code.
+
+To maintain a clean commit history, features and fixes are usually squash-merged,
+but not hot-fixes; see [Merging](#merging) for the complete details.
+
+### main
+
+The `main` branch contains stable, fully-tested code. It should only be merged
+into via CI/CD flows, and only from the `dev` branch or a hot-fix branch.
+
+Merges into `main` **must always** be plain-merged, **never** squashed; see
+[Merging](#merging) for a thorough-going explanation of this rationale.
+
+## Style
+
+### EditorConfig
```

Comment:
Seeing this section made me think the initial setup should include:

- `.gitignore`
- `.gitattributes`
- `.vscode/extensions.json`
    - For now I suppose it should just recommend the `[EditorConfig for VS Code]` extension.
- `.editorconfig`